Automattic/jetpack
# Add Jetpack Starter post-purchase flow to recommendations

## Proposed changes:
After we recently launched the new Jetpack Starter plan, we want to support it in our post-purchase flow in Jetpack recommendations module.

## Testing instructions:
* Checkout this PR and run your local jetpack dev instance
* Ensure you don't have JP Complete, Security, or Starter plans attached to the site
* "Reset Options (dev only)" - you can find it in footer on `/wp-admin/admin.php?page=jetpack`
* Purchase JP Starter plan and attach it to the site
* You should be redirected to `/wp-admin/admin.php?page=jetpack#/recommendations` and see JP Starter welcome card:
<img width="1082" alt="CleanShot 2023-05-12 at 15 22 38@2x" src="https://github.com/Automattic/jetpack/assets/8419292/30276bac-1dd7-4edf-9a63-f7400e4ecff0">


* You should be able to complete the flow and see the following summary:
<img width="1057" alt="CleanShot 2023-05-12 at 15 20 32@2x" src="https://github.com/Automattic/jetpack/assets/8419292/c6a2a31e-94f6-42c4-99f7-b70e1514c366">
